scripts/pre-push-tests.sh

```
@@ -0,0 +1,77 @@
+if [ ! -s "$changed_files" ]; then
+  echo "[pre-push] Nenhuma mudanca detectada. Pulando testes."
+  exit 0
+fi
+
+run_front=false
+run_back=false
+run_all=false
+
+while IFS= read -r file; do
+  case "$file" in
+    central-pet-front/*)
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 45.

(SC2221)